.github/workflows/robustness-template-arm64.yaml

```diff
@@ -26,7 +26,9 @@ jobs:
   test:
     timeout-minutes: 210
     runs-on: ${{ fromJson(inputs.runs-on) }}
-    container: golang:1.19-bullseye
+    container:
+      image: golang:1.19-bullseye
+      options: --privileged --cap-add SYS_ADMIN --device /dev/fuse
```

Comment:
With privileged, if the engine is docker, by default the engine will mount all the device from host into container. And we don't need to add cap SYS_ADMIN because it's ready privileged.

Reply:
Great catch, and yes it's `docker` on the two self hosted runners at the moment. Updated thanks!